core/domain/classifier_domain.py

```diff
@@ -365,3 +365,71 @@ def validate(self):
                 raise utils.ValidationError(
                     'Expected answers to be a list, received %s' %
                     grouped_answers['answers'])
+
+
+class ClassifierExplorationMapping(object):
+    """Domain object for a classifier-exploration mapping model.
+
+    A Classifier-Exploration mapping is a one-to-one relation between the
+    attributes in an exploration to the classifier model it needs to utilise.
+    The mapping is from <exp_id, exp_version, state_name> to the classifier_id.
+
+    Attributes:
+        mapping_id: str. The unique id of the classifier-exploration mapping.
```

Comment:
@prasanna08 . Doesn't the Domain layer usually reflect the model? Thats why I kept Domain layer with a mapping_id attribute instead of three attributes (exp_id, exp_version, state_name). So, when the helper function inside classifier_services receives the three attributes as parameters, I generated the mapping_id within that function and the used get_by_id. How do you think the Domain layer should be?